ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -0,0 +1,251 @@
+    aggregateByCreatedTime,
+    aggregateByImageSha,
+    defaultCVESortFields,
+    getScoreVersionsForTopCVSS,
+    sortCveDistroList,
+} from 'Containers/Vulnerabilities/WorkloadCves/sortUtils';
+import { VulnerabilityExceptionScope } from 'services/VulnerabilityExceptionService';
+import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
+
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+// @TODO: Move these files up to a common directory and move the types used in these files as well
+import TableErrorComponent from 'Containers/Vulnerabilities/WorkloadCves/components/TableErrorComponent';
+import SeverityCountLabels from '../../WorkloadCves/components/SeverityCountLabels';
+import CvssTd from '../../WorkloadCves/components/CvssTd';
+import DateDistanceTd from '../../WorkloadCves/components/DatePhraseTd';
+
+type RequestCVEsTableProps = {
+    cves: string[];
+    scope: VulnerabilityExceptionScope;
+    expandedRowSet: SetResult<string>;
+};
+
+function getImageScope(scope: VulnerabilityExceptionScope): string {
+    if (
+        scope.imageScope.registry === '.*' &&
+        scope.imageScope.remote === '.*' &&
+        scope.imageScope.tag === '.*'
+    ) {
+        return '';
+    }
+    if (scope.imageScope.tag === '.*') {
+        return `${scope.imageScope.registry}/${scope.imageScope.remote}`;
+    }
+    return `${scope.imageScope.registry}/${scope.imageScope.remote}:${scope.imageScope.tag}`;
+}
+
+function RequestCVEsTable({ cves, scope, expandedRowSet }: RequestCVEsTableProps) {
+    const { page, perPage, setPage } = useURLPagination(20);
+    const { sortOption, getSortParams } = useURLSort({
+        sortFields: defaultCVESortFields,
+        defaultSortOption: CVEsDefaultSort,
+        onSort: () => setPage(1),
+    });
+
+    const queryObject = {
+        CVE: cves.join(','),
+        Image: getImageScope(scope),
+    };
+
+    const query = getRequestQueryStringForSearchFilter(queryObject);
+
+    const { error, loading, data } = useQuery<CVEListQueryResult>(cveListQuery, {
+        variables: {
+            query,
+            pagination: {
+                offset: (page - 1) * perPage,
+                limit: perPage,
+                sortOption,
+            },
+        },
+    });
+
+    if (loading && !data) {
+        return (
+            <Bullseye>
+                <Spinner isSVG />
+            </Bullseye>
+        );
+    }
+
+    if (error) {
+        return (
+            <PageSection variant="light">
+                <TableErrorComponent
+                    error={error}
+                    message="An error occurred. Try refreshing again"
+                />
+            </PageSection>
+        );
+    }
+
+    return (
+        <PageSection variant="light">
+            <Flex direction={{ default: 'column' }}>
+                <Title headingLevel="h2">{data?.imageCVEs.length || 0} results found</Title>
+                <TableComposable variant="compact">
```

Comment:
This table is very close to identical to the one used on the Workload CVE list page. Do you think it is worth revisiting later to see if we combine the two?

The former already has a handful of customization options as props, but I worry that adding more will just make the code difficult to parse. Maybe there is a better way we could use composition to make it reusable while avoiding the complexity.

Reply:
Yea, I agree that adding more props sometimes can lead to unintended consequences and/or more effort trying to refactor in the future. I can leave a TODO comment here about going back and considering how to make it more reusable (like we did for the Autocomplete filter component)